src/gui/QvisColorTableWindow.C

```diff
@@ -2126,11 +2181,51 @@ QvisColorTableWindow::deleteColorTable()
         return;
     }
 
+    if (nameListBox->topLevelItemCount() == 0)
+    {
+        QString tmp;
+        tmp = tr("Not able to delete a color table; there are no color tables"
+                 " to delete.");
+        Error(tmp);
+        return;
+    }
+
     // Get the index of the currently selected color table and tell the viewer
     // to remove it from the list of color tables.
-    std::string ctName = nameListBox->currentItem()->text(0).toStdString();
+    if (QTreeWidgetItem *item = nameListBox->currentItem())
+    {
+        std::string ctName = item->text(0).toStdString();
+        // TODO remove the `const_cast` on develop; this issue has been solved there.
```

Comment:
Is this `TODO` comment still relevant?

Reply:
Yes; I will address it when I merge these changes into develop. For a variety of reasons, there is extra work that will need to be done to make these changes work on develop, so I will work on that after the 3.3.1 deadline. There'll be a new PR for that.